meterfs/meterfs.h

```diff
@@ -15,11 +15,15 @@
 #define _METERFS_H_
 
 #include <sys/rb.h>
+#include <storage/storage.h>
 
 #define MAX_NAME_LEN 8
 
 
-enum { meterfs_allocate = 0, meterfs_resize, meterfs_info, meterfs_chiperase };
+enum { meterfs_allocate = 0,
+	meterfs_resize,
+	meterfs_info,
+	meterfs_chiperase };
```

Comment:
Leave this as is, we have slight problems still with clang format. You can add `/* clang-format off */` and `on` if you wish